pytorch_lightning/loggers/tensorboard.py

```diff
@@ -109,8 +109,15 @@ def log_hyperparams(self, params: Union[Dict[str, Any], Namespace]) -> None:
                 " hyperparameter logging."
             )
         else:
+            # Passing only those params which tensorboard allows here:
+            # https://github.com/pytorch/pytorch/blob/master/torch/utils/tensorboard/summary.py#L134
+            from six import string_types
             from torch.utils.tensorboard.summary import hparams
-            exp, ssi, sei = hparams(params, {})
+            tensorboard_params = {}
+            for k, v in params.items():
+                if isinstance(v, (int, float, string_types, bool, torch.Tensor)):
```

Comment:
i think this is no longer necessary because non-primitive hparams are already converted to string (see master branch)

Reply:
agree